packages/sit-onyx/src/components/examples/GridPlayground/EditGridElementDialog/EditGridElementDialog.ct.tsx

```diff
@@ -0,0 +1,90 @@
+import { expect, test } from "../../../../playwright/a11y";
+import EditGridElementDialog, { type GridElementConfig } from "./EditGridElementDialog.vue";
+
+test("should behave correctly", async ({ mount, makeAxeBuilder, page }) => {
+  const submitEvents: GridElementConfig[] = [];
+  let closeEvents = 0;
+
+  // ARRANGE
+  await page.setViewportSize({ width: 512, height: 768 });
+  await mount(
+    <EditGridElementDialog
+      open
+      onSubmit={(element) => submitEvents.push(element)}
+      onClose={() => closeEvents++}
+    />,
+  );
+  const dialog = page.getByRole("dialog", { name: "Column configuration" });
+
+  // ACT
+  const accessibilityScanResults = await makeAxeBuilder()
+    .disableRules(
+      // TODO: remove when contrast issues are fixed in https://github.com/SchwarzIT/onyx/issues/410
+      ["color-contrast"],
+    )
+    .analyze();
+
+  // ASSERT
+  expect(accessibilityScanResults.violations).toEqual([]);
+  await expect(page).toHaveScreenshot("default.png");
+  await expect(dialog).toBeVisible();
+
+  const columnCountStepper = dialog.getByLabel("Default number of columns");
+  await expect(columnCountStepper).toBeFocused();
+  await expect(columnCountStepper).toHaveAttribute("min", "1");
```

Comment:
imho, for those generically named `should behave correctly` tests, it's really helpful to use the 2nd expect parameter to give info about what you're checking so if it fails, you are a bit more descriptive.

so give all/most expects something like

```suggestion
  await expect(columnCountStepper, 'stepper allows at least 1').toHaveAttribute("min", "1");
```
is quite helpful :)


of course, as this is more of a demo component that we (probably) won't touch too often, you can ignore this comment for this PR :)

Reply:
Nice idea! I updated it for a few expects that might not be 100% self explanatory :)